conans/client/tools/system_pm.py

```diff
@@ -126,6 +126,64 @@ def install(self, packages, update=True, force=False, arch_names=None):
 
+    def install_packages(self, packages, update=True, force=False, arch_names=None):
+        """ Get the system package tool install command and install all packages and/or variants.
+            Inputs:
+            "pkg-variant1"  # (1) Install only one package
+            ["pkg-variant1", "otherpkg", "thirdpkg"] # (2) All must be installed
+            [("pkg-variant1", "pkg-variant2"), "otherpkg", "thirdpkg"] # (3) Install only one variant
+                                                                             and all other packages
+            "pkg1 pkg2", "pkg3 pkg4" # (4) Invalid
+            ["pkg1 pkg2", "pkg3 pkg4"] # (5) Invalid
+
+        :param packages: Supports multiple formats (string,list,tuple). Lists and tuples into a list
+        are considered variants and is processed just like self.install(). A list of string is
+        considered a list of packages to be installed (only not installed yet).
+        :param update: Run update command before to install
+        :param force: Force installing all packages, including all installed.
+        :param arch_names: Package suffix/prefix name used by installer tool e.g. {"x86_64": "amd64"}
+        :return: None
+        """
+        packages = [packages] if isinstance(packages, str) else list(packages)
+        # only one (first) variant will be installed
+        list_variants = list(filter(lambda x: isinstance(x, (tuple, list)), packages))
+        for variants in list_variants:
+            self.install(variants, update=update, force=force, arch_names=arch_names)
+        # all packages will be installed
+        packages = list(filter(lambda x: not isinstance(x, (tuple, list)), packages))
+
+        if [pkg for pkg in packages if " " in pkg]:
+            raise ConanException("Each string must contain only one package to be installed. "
+                                 "Use a list instead e.g. ['foo', 'bar'].")
+
+        packages = self._get_package_names(packages, arch_names)
+
+        mode = self._get_sysrequire_mode()
+
+        if mode in ("verify", "disabled"):
+            # Report to output packages need to be installed
+            if mode == "disabled":
+                self._output.info("The following packages need to be installed:\n %s"
+                                  % "\n".join(packages))
+                return
+
+            if mode == "verify" and not self._installed(packages):
+                self._output.error("The following packages need to be installed:\n %s"
+                                   % "\n".join(packages))
+                raise ConanException("Aborted due to CONAN_SYSREQUIRES_MODE=%s. "
+                                     "Some system packages need to be installed" % mode)
+
+        to_be_installed = self._to_be_installed(packages)
+        if not force and not to_be_installed:
+            return
+        # install only what is not installed yet
+        packages = packages if force else to_be_installed
```

Comment:
Do not run all the operations in `self._to_be_installed` if we are going to `force` the installation

```suggestion
        packages = packages if force else self._to_be_installed(packages)
        if not force and not packages:
            return
```

Reply:
Done.